src/main/java/com/qaprosoft/carina/demo/gui/components/ParagraphElement.java

```diff
@@ -0,0 +1,33 @@
+package com.qaprosoft.carina.demo.gui.components;
+
+import com.qaprosoft.carina.core.foundation.webdriver.decorator.ExtendedWebElement;
+import com.qaprosoft.carina.core.gui.AbstractUIObject;
+import org.openqa.selenium.SearchContext;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ParagraphElement extends AbstractUIObject {
+
+    @FindBy(xpath = ".//a")
+    private List<ExtendedWebElement> linkElements;
+
+    private List<String> linkElementsString = new ArrayList<>();
```

Comment:
You don't need it List as class field.It's better to declare this list in getParagraphGlossaryElements() method.